plugins/uberdev/skills/testers-pipeline/aggregate.py

```
@@ -123,7 +125,38 @@ def main() -> int:
     with open(args.out, "w") as fh:
         yaml.safe_dump(out, fh, default_flow_style=False, sort_keys=False)
     print(f"aggregated {len(findings)} findings into {args.out}", file=sys.stderr)
-    return 0
+
+    plugin_root = os.environ.get("CLAUDE_PLUGIN_ROOT")
+    if not plugin_root:
+        print("error: CLAUDE_PLUGIN_ROOT unset; cannot source lib/rate-cap-audit.sh", file=sys.stderr)
+        return 2
+    audited_path = f"{args.out}.audited"
+    audit_proc = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)